assets/javascripts/discourse/lib/ai-streamer.js

```diff
@@ -113,13 +113,16 @@ class PostUpdater extends StreamUpdater {
     await this.setCooked(cookedElement.innerHTML);
   }
 
-  async setCooked(value) {
-    this.post.set("cooked", value);
+  async setCooked(cooked) {
+    this.post.set("cooked", cooked);
 
     const oldElement = this.postElement.querySelector(".cooked");
 
-    await loadScript("/javascripts/diffhtml.min.js");
-    window.diff.innerHTML(oldElement, value);
+    // TODO: use `morphInner` once version 0.0.16 of morphlex is out
+    const newElement = oldElement.cloneNode(false);
+    newElement.innerHTML = cooked;
+
+    (await import("morphlex")).morph(oldElement, newElement);
```

Comment:
Huh - does this actually work? I didn't think plugin JS was run through webpack... so I don't think the `import()` will be transpiled into something functional 🤔